account_vat_period_end_statement/__openerp__.py

```diff
@@ -33,16 +34,20 @@
         "report",
         "l10n_it_fiscalcode",
         ],
-    "author": "Agile Business Group, Odoo Community Association (OCA)",
+    "author": """
+        Agile Business Group, Odoo Community Association (OCA),
+        LinkIt Spa
+        """,
```

Comment:
Questo introdurrebbe diversi spazi prima di 'Agile' e prima di 'LinkIt'.
Lo puoi riscrivere in questo modo:
```
    "author": "Agile Business Group, Odoo Community Association (OCA)"
              ", LinkIt Spa",
```